pkg/transport/transport_test.go

```diff
@@ -8,11 +8,19 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
+	"storj.io/storj/pkg/provider"
 	proto "storj.io/storj/protos/overlay"
 )
 
 func TestDialNode(t *testing.T) {
-	oc := Transport{}
+	ca, err := provider.NewCA(context.Background(), 12, 4)
```

Comment:
for tests, to make sure no one gets into the habit of using context.Background() in code, we've been defining a global `var ctx = context.Background()` variable so all the tests can just use `ctx`